src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -367,5 +382,20 @@ public static void MinimumBundleVersionNotSatisfied(this ILogger logger, string
         {
             _minimumBundleVersionNotSatisfied(logger, bundleId, bundleVersion, minimumVersion, minimumVersion, null);
         }
+
+        public static void ValidateHostJsonZipIssue(this ILogger logger, string path)
```

Comment:
Does any log written here go to both AppInsights and Console?

Reply:
yes, to both, console and AppInsights.